aldryn_newsblog/cms_appconfig.py

```diff
@@ -73,7 +73,14 @@ class NewsBlogConfig(TranslatableModel, AppHookConfig):
         help_text=_('When grouping page numbers, this determines how many '
                     'pages are visible on each side of the active page.'),
     )
-
+    exclude_featured = models.PositiveSmallIntegerField(
+        _('Excluded featured articles count'),
+        blank=True,
+        default=0,
+        help_text=_('If you are using featured articles plugin on article list '
+                    'page you might want to avoid duplicated articles. '
+                    'This option allows you to exclude first N of them.'),
+    )
```

Comment:
Suggested text:

> If you are using the Featured Articles plugin on the article list view, you may prefer to exclude featured articles from the article list itself to avoid duplicates. To do this, enter the same number here as in your Featured Articles plugin.